components/mixins/profile/profile-tax-profile-details.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile/profile-tax-profile-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Personal Finance Details",
```

Comment:
Should this be more elaborative like "Personal Tax Profile" or something like that ?

Reply:
I'll rename it with the next commit.